src/test/migrations/reset.ts

```diff
@@ -0,0 +1,65 @@
+import 'tsconfig-paths/register'
+import 'dotenv/config'
+
+import * as fs from 'fs'
+import * as path from 'path'
+import * as readline from 'readline'
+
+import { DB } from 'server/db'
+import { Logger } from 'server/utils/logger'
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+const _confirmReset = (): Promise<boolean> => {
```

Comment:
Added a user confirm:
After running command, user needs to write `reset` in the terminal to avoid running this by mistake (mainly to avoid resetting the db if the user didn't want to run this command)